tests/unit/test_models.py

```diff
@@ -26,37 +27,49 @@ def test_simulate_without_build_model(self):
         ):
             model.simulateS1(None, None)
 
+    @pytest.mark.parametrize(
+        "model",
+        [pybop.lithium_ion.SPM(), pybop.lithium_ion.SPMe(), pybop.empirical.Thevenin()],
+    )
     @pytest.mark.unit
-    def test_predict_without_pybamm(self):
-        # Define model
-        model = pybop.lithium_ion.SPM()
+    def test_predict_without_pybamm(self, model):
         model._unprocessed_model = None
 
         with pytest.raises(ValueError):
             model.predict(None, None)
 
+    @pytest.mark.parametrize(
+        "model",
+        [pybop.lithium_ion.SPM(), pybop.lithium_ion.SPMe(), pybop.empirical.Thevenin()],
+    )
     @pytest.mark.unit
-    def test_predict_with_inputs(self):
-        # Define SPM
-        model = pybop.lithium_ion.SPM()
+    def test_predict_with_inputs(self, model):
+        # Define inputs
         t_eval = np.linspace(0, 10, 100)
-        inputs = {
-            "Negative electrode active material volume fraction": 0.52,
-            "Positive electrode active material volume fraction": 0.63,
-        }
+        if isinstance(model, (pybop.lithium_ion.SPM, pybop.lithium_ion.SPMe)):
+            inputs = {
+                "Negative electrode active material volume fraction": 0.52,
+                "Positive electrode active material volume fraction": 0.63,
+            }
+            signal = "Terminal voltage [V]"
```

Comment:
Since an early release of PyBaMM last year, "Voltage [V]" == "Terminal Voltage [V]". To remain consistent we can move towards using "Voltage [V]" for all signal items.

```suggestion
            signal = "Voltage [V]"
```